packages/compass-e2e-tests/tests/in-use-encryption.test.ts

```diff
@@ -7,35 +7,58 @@ import { MONGODB_VERSION } from '../helpers/compass';
 import * as Selectors from '../helpers/selectors';
 import { getFirstListDocument } from '../helpers/read-first-document-content';
 import { MongoClient } from 'mongodb';
+import path from 'path';
+
+import delay from '../helpers/delay';
+
+import { promises as fs } from 'fs';
+
+import { LOG_PATH } from '../helpers/compass';
 
 const CONNECTION_HOSTS = 'localhost:27091';
-const CONNECTION_STRING = `mongodb://${CONNECTION_HOSTS}`;
+const CONNECTION_STRING = `mongodb://${CONNECTION_HOSTS}/`;
 
-describe('FLE2', function () {
-  before(function () {
-    if (
-      semver.lt(MONGODB_VERSION, '6.0.0-rc0') ||
-      process.env.MONGODB_USE_ENTERPRISE !== 'yes'
-    ) {
-      return this.skip();
+const readSavedConnectionsFolder = async (compass: Compass) => {
+  if (compass.userDataPath) {
+    try {
+      const connections = await fs.readdir(
+        path.join(compass.userDataPath, 'Connections')
+      );
+
+      console.log('Compass connections:');
+      console.log(connections);
+    } catch (err) {
+      console.error('Error during reading connections:');
+      console.error(err);
     }
-  });
+  }
+};
 
-  describe('when fleEncryptedFieldsMap is not specified while connecting', function () {
-    const databaseName = 'db-for-fle';
-    const collectionName = 'my-encrypted-collection';
+describe('FLE2', function () {
+  describe('server version gte 4.2.20 and not a linux platform', function () {
+    const databaseName = 'fle-test';
+    const collectionName = 'my-another-collection';
     let compass: Compass;
     let browser: CompassBrowser;
 
     before(async function () {
+      if (
+        semver.lt(MONGODB_VERSION, '4.2.20') ||
+        process.env.MONGODB_USE_ENTERPRISE !== 'yes' ||
+        process.platform === 'linux'
```

Comment:
Is there a ticket for this? If so, can you link it in a comment (and link the ticket as related to the one for this PR)? `// TODO(COMPASS-XXXX): Fix this on Linux`